### ISSUE TITLE
[BUG] enterprise: Input should be a valid string

### ISSUE THREAD

**Issue by Kostavro** [echoboomer/incident-bot#593]
**Describe the bug**
When doing various actions through Slack, they fail (like for example `List Incidents`). In the app's logs I see:
```
incident-bot-5445c69868-pvgfx incident-bot Error: 1 validation error for SlackBlockActionsResponse
incident-bot-5445c69868-pvgfx incident-bot enterprise
incident-bot-5445c69868-pvgfx incident-bot   Input should be a valid string [type=string_type, input_value={'id': 'X1X1X1', 'name': 'XXXXX'}, input_type=dict]
incident-bot-5445c69868-pvgfx incident-bot     For further information visit https://errors.pydantic.dev/2.9/v/string_type
incident-bot-5445c69868-pvgfx incident-bot Traceback (most recent call last):
incident-bot-5445c69868-pvgfx incident-bot   File "/usr/lib/python3.12/site-packages/slack_bolt/listener/thread_runner.py", line 120, in run_ack_function_asynchronously
incident-bot-5445c69868-pvgfx incident-bot     listener.run_ack_function(request=request, response=response)
incident-bot-5445c69868-pvgfx incident-bot   File "/usr/lib/python3.12/site-packages/slack_bolt/listener/custom_listener.py", line 50, in run_ack_function
incident-bot-5445c69868-pvgfx incident-bot     return self.ack_function(
incident-bot-5445c69868-pvgfx incident-bot            ^^^^^^^^^^^^^^^^^^
incident-bot-5445c69868-pvgfx incident-bot   File "/home/nonroot/incidentbot/slack/handler.py", line 545, in handle_static_action
incident-bot-5445c69868-pvgfx incident-bot     parsed_body = SlackBlockActionsResponse(**body)
incident-bot-5445c69868-pvgfx incident-bot                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
incident-bot-5445c69868-pvgfx incident-bot   File "/usr/lib/python3.12/site-packages/pydantic/main.py", line 212, in __init__
incident-bot-5445c69868-pvgfx incident-bot     validated_self = self.__pydantic_validator__.validate_python(data, self_instance=self)
```
where `X1X1X1` is the Slack id of my workspace, and `XXXXX` is the name of my Slack workspace.

**Version**
v2.0.0

**Expected behavior**
I expect the commands to work.

**Screenshots**
-

**Environment (please complete the following information):**
 - Platform: Kubernetes

**Additional context**
-

**Comment by echoboomer** [comment]
Ok, that's a bug on the custom Slack types. I should be able to resolve that fairly easily, thanks for reporting it.

**Comment by echoboomer** [comment]
> Thanks @echoboomer . I just noticed another one - related to the db, leaving here, but I can also open a new issue if you wish to track it separately:
> 
> ```
> incident-bot-6d4d884454-ndz2q incident-bot AttributeError: 'dict' object has no attribute 'status'
> incident-bot-6d4d884454-ndz2q incident-bot Error: 'dict' object has no attribute 'status'
> incident-bot-6d4d884454-ndz2q incident-bot Traceback (most recent call last):
> incident-bot-6d4d884454-ndz2q incident-bot   File "/usr/lib/python3.12/site-packages/slack_bolt/listener/thread_runner.py", line 120, in run_ack_function_asynchronously
> incident-bot-6d4d884454-ndz2q incident-bot     listener.run_ack_function(request=request, response=response)
> incident-bot-6d4d884454-ndz2q incident-bot   File "/usr/lib/python3.12/site-packages/slack_bolt/listener/custom_listener.py", line 50, in run_ack_function
> incident-bot-6d4d884454-ndz2q incident-bot     return self.ack_function(
> incident-bot-6d4d884454-ndz2q incident-bot            ^^^^^^^^^^^^^^^^^^
> incident-bot-6d4d884454-ndz2q incident-bot   File "/home/nonroot/incidentbot/slack/modals.py", line 115, in update_home_tab
> incident-bot-6d4d884454-ndz2q incident-bot     open_incidents = BlockBuilder.incident_list(incidents=database_data)
> incident-bot-6d4d884454-ndz2q incident-bot                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> incident-bot-6d4d884454-ndz2q incident-bot   File "/home/nonroot/incidentbot/slack/messages.py", line 667, in incident_list
> incident-bot-6d4d884454-ndz2q incident-bot     if incident.status
> ```

Do you have any open incidents when you encounter this error? Are you seeing this error when you try to use the list command or on the app home page or both?

**Comment by Kostavro** [comment]
This (the `incident.status`) error happens when I open the incidentbot's Slack user's profile - where it shows the `Declare Incident` button, and sections: `Creating Incidents`, `Documentation and Learning Materials` and `Incidents`. `Incidents` is empty and says `No open incidents`.

**Comment by echoboomer** [comment]
I was just able to reproduce. Thanks for reporting this as well, I will push up a fix along with the other issue too.

**Comment by Kostavro** [comment]
Thank you very much 😄

**Comment by echoboomer** [comment]
@Kostavro just released `v2.0.4`, if you have time please try bumping to that chart version and seeing if your issues are resolved. Thanks.

**Comment by Kostavro** [comment]
Hey @echoboomer , just tested the latest changes - both issues reported above are now solved and functionality works as intended! Thank you again for the quick resolution of the bugs!
